ci/telemetry_autoscaling.yml

```
@@ -0,0 +1,47 @@
+---
+- name: Telemetry autoscaling tests
+  hosts:  "{{ cifmw_target_hook_host | default('localhost')  }}"
+  gather_facts: false
```

This is needed for ansible_env to be available. this will cause the job to fail again next time.
```suggestion
  gather_facts: true
```